client/my-sites/marketplace/types.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { YOAST_PREMIUM, YOAST_FREE } from '@automattic/calypso-products';
+
+export interface IProductDefinition {
+	/**
+	 * defaultPluginSlug : Some plugins may not be available in the calypso product library so we specify a default plugin that is accessible in the product library
+	 * so that relevant details can be shown
+	 * */
+	defaultPluginSlug: string;
+	pluginsToBeInstalled: string[];
+	isPurchasableProduct: boolean;
+}
+
+export interface IProductCollection {
+	[ YOAST_PREMIUM ]: IProductDefinition;
+	[ YOAST_FREE ]: IProductDefinition;
+}
+
+/**
+ * A set of logical product groups, grouped by actual products, to be shown in one product (group) page
+ * i.e. : YOAST_PREMIUM, YOAST_FREE are 2 products that belong to the product group YOAST
+ * */
+export const YOAST = 'YOAST';
```

Comment:
Does this belong somewhere else, considering that it is not (and is not consumed as) a type? Perhaps it should go to the constants file or `marketplace.ts`?

Reply:
If we move those definitions to a marketplace-products.ts perhaps maybe this can also live there.